components/Hero.js

```diff
@@ -11,10 +11,12 @@ import Box from './Box';
 const Hero = Box.extend`
   background: ${props =>
     props.colorful
-      ? 'linear-gradient(90deg, #014C90 0%, #4EAD81 100%)'
+      ? 'linear-gradient(135deg, #004b91 0%, #5abc7f 100%)'
```

Comment:
Same comment as above - call these `props.theme.gradients.gradient4.start` and `props.theme.gradients.gradient4.end` or something similar?